### PR TITLE
Feature/atat 7198

### PR DESCRIPTION
**Pull request by DGabriel-CCPO**

### CODE REVIEW

File: src/router/resolvers/index.ts [hunk 1]
```diff
@@ -49,6 +50,16 @@ export const CurrentContractRouteResolver = (current: string): string => {
...
   }
 
-  return routeNames.Performance_Requirements; 
+  return routeNames.Performance_Requirements;
 };
 
+
+export const PIIRecordResolver = (current: string): string => {
+  const systemOfRecord = PIIRecord.PIIRecordIncluded;
+  console.log(systemOfRecord)
```
Comment (ericyoungquist-ccpo) [inline src/router/resolvers/index.ts:59]: remove console log

File: src/router/resolvers/index.ts [hunk 2]
```diff
@@ -49,6 +50,16 @@ export const CurrentContractRouteResolver = (current: string): string => {
...
+  // if system of record will be included, route to system of records page
+  if (current === routeNames.PII && systemOfRecord) {
+    return routeNames.PIIRecord;
+  }
+  return 'Public_Disclosure_of_Information';
```
Comment (ericyoungquist-ccpo) [inline src/router/resolvers/index.ts:64]: is there a `routeNames` equivalent for this?

File: src/router/stepper.ts [hunk 3]
```diff
@@ -29,13 +29,16 @@ import WillGovtEquipBeFurnished from "../steps/GovtFurnishedEquipment/WillGovtEq
...
 import PII from "../steps/OtherContractConsiderations/PII.vue";
+import PIIRecord from "../steps/OtherContractConsiderations/PIIRecord.vue";
 
 // other
 import ValidatorsExample from "../validation/ValidatorsExample.vue";
 
 // route resolves
-import { AcorsRouteResolver, CustodianRouteResolver, CurrentContractRouteResolver } from "./resolvers";
+
+import { AcorsRouteResolver, CustodianRouteResolver, CurrentContractRouteResolver, PIIRecordResolver } from "./resolvers";
```
Comment (ericyoungquist-ccpo) [inline src/router/stepper.ts:41]: stack these please... getting long

File: src/router/stepper.ts [hunk 4]
```diff
@@ -273,13 +277,23 @@ export const stepperRoutes: Array<StepperRouteConfig> = [
     name: "Other_Contract_Considerations",
```
Comment (ericyoungquist-ccpo) [inline src/router/stepper.ts:277]: pls add to `routeNames`

File: src/steps/OtherContractConsiderations/PII.vue [hunk 5]
```diff
@@ -104,11 +104,10 @@ import {RadioButton} from "../../../types/Global";
 })
 
 export default class PII extends Vue {
-  private selectedPIIOption = "";
   private pIIOptions: RadioButton[] = [
     {
       id: "YesPII",
-      label: "Yes. This contract action will include a system of records with PII.",
+      label: "Yes. This contract action will include a system of records with PIIRecordStore.",
```
Comment (ericyoungquist-ccpo) [inline src/steps/OtherContractConsiderations/PII.vue:110]: looks like a search/replace oops there

File: src/steps/OtherContractConsiderations/PII.vue [hunk 6]
```diff
@@ -117,5 +116,17 @@ export default class PII extends Vue {
...
+  public get selectedPIIOption(): string {
+    const included = PIIRecord.PIIRecordIncluded;
+    if (included !== null) {
+      return included ? "Yes" : "No";
+    }
+    return "";
+  }
+
+  public set selectedPIIOption(value: string) {
+    PIIRecord.setPIIRecord(value === "Yes" ? true : false);
```
Comment (ericyoungquist-ccpo) [inline src/steps/OtherContractConsiderations/PII.vue:129]: I had it that way originally... shorter is `(value === "Yes")` and forget the true/false - it's unneeded

**Review by ericyoungquist-ccpo — CHANGES_REQUESTED** [review]

File: src/router/resolvers/index.ts (hunk 2 above)
Comment (DGabriel-CCPO) [reply src/router/resolvers/index.ts:64]: not yet

File: src/steps/OtherContractConsiderations/PII.vue (hunk 5 above)
Comment (DGabriel-CCPO) [reply src/steps/OtherContractConsiderations/PII.vue:110]: Thank you for catching that.

File: src/router/resolvers/index.ts (hunk 2 above)
Comment (DGabriel-CCPO) [reply src/router/resolvers/index.ts]: just added it.

**Review by ericyoungquist-ccpo — APPROVED** [review]

**Pull request merged**